test/ng/supportSpec.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
Why in `test/helpers`? `test/helpers/` contains helpers for tests, not actual tests.